src/data/content/design-system/componenti/megamenu.yaml

```diff
@@ -86,14 +85,15 @@ tabs:
             title: Come usarlo
             headingLevel: 2
             text: |
-              - Per dare contesto al  
+              - Per permettere all’utente di comprendere l’organizzazione dei contenuti e di saltare direttamente a pagine di livello inferiore.
```

Comment:
In merito alla forma per il "Come usarlo", nel caso di [Card](https://github.com/italia/designers.italia.it/pull/1181/commits/ee045f7799ad97f623c0a9377637ae9bbee5c197) sto usando un approccio diverso:

**Azione su come usarlo(grassetto)** Testo di spiegazione che può essere un paragrafo di una o due righe.

Esempio:

**Rendi le card azionabili.** Poiché sono una sintesi di informazioni più dettagliate, ogni singola card deve puntare a questi contenuti con un link.
---

La forma verbale diretta della seconda persona singolare aiuta, secondo me, il fluire del contenuto creando raccordo tra titoli e paragrafi. 

Cosa ne pensate? @Fupete @danielaiozzo

Reply:
In generale bisogna rimanere coerenti con quanto già fatto nelle altre schede componente fatte in questo giro. Non cambiamo in corsa o si crea frammentazione. 
Corcordo sulla seconda persona, qui mi sa che non è ancora passata Daniela per i fix.

Reply:
Intanto: @danielaiozzo quando hai tempo fai pure l'ultimo giro di review & fix così mergiamo anche questa 🙏